templates/vue/src/components/NodeModal.vue

```diff
@@ -144,17 +144,16 @@
               </combobox>
             </b-form-group>
             <b-form-group
-              v-show="node.mediaType === 'url-embed'"
+              v-if="node.mediaType === 'url-embed'"
               label="External Link"
             >
-              <b-form-input
-                id="node-embed-media-duration"
+              <file-upload
                 v-model="node.typeData.mediaURL"
-                placeholder="Enter embed link (starting with http)"
-                required
+                input-id="node-embed-media-duration"
```

Comment:
We can probably remove this and the `inputId` prop altogether from the component.